app/api_clients/control_panel_api.js

```diff
@@ -42,31 +42,39 @@ class DjangoError extends APIError {
   constructor(error) {
     super(error);
     this.original_message = this.message;
+    const pattern = /500 - "(.*)Request Method:/;
+    const match = this.original_message.match(pattern);
+    this.message = match[1]
+      .trim()
+      .replace(/\\n/g, '<br>\n');
   }
 
   static match(error) {
     return error.message && error.message.indexOf('Traceback:') >= 0;
   }
 
   get python_traceback() {
-    let pattern = /\\nTraceback:\s+\\n\\n(.*)\\n\\nException Type/;
-    const match = this.original_message.match(pattern);
+    if (!this.original_message) {
+      return [];
+    }
+    let pattern = /Traceback:(.*)Exception Type/;
+    let match = this.original_message.match(pattern);
     const traceback = match[1]
       .replace(/\\n/g, '\n')
+      .trim()
       .replace(/\\"/g, '"')
       .split(/\n\n/);
 
     return traceback.map((line) => {
       pattern = /^File "([^"]+)" in ([^\n]+)[^0-9]+([0-9]+). ([^\n]+)/;
-      const [file, func, line_no, code] = line.match(pattern).slice(1);
-      return { file, func, line_no, code };
-    });
-  }
+      match = line.match(pattern);
+      if (match) {
+        const [file, func, line_no, code] = match.slice(1);
+        return { file, func, line_no, code };
+      }
 
-  get message() {
```

Comment:
This `message()` is not used anywhere right?

Reply:
`get message()` has been replaced by `this.message` on line 47